framework/helpers/BaseFileHelper.php

```diff
@@ -162,6 +162,10 @@ public static function getMimeType($file, $magicFile = null, $checkExtension = t
             throw new InvalidConfigException('The fileinfo PHP extension is not installed.');
         }
 
+        if (PHP_VERSION_ID >= 80100) {
+            return static::getMimeTypeByExtension($file, $magicFile);
```

Comment:
Doesn't seem to be correct. Maybe it's a good idea to have a test that is broken first?